src/ListView.php

```diff
@@ -182,14 +239,16 @@ protected function renderItem(array|object $data, mixed $key, int $index): strin
         }
 
         if ($this->itemView instanceof Closure) {
-            $content = (string) call_user_func($this->itemView, $data, $key, $index, $this);
+            $content = (string)call_user_func($this->itemView, $data, $key, $index, $this);
         }
 
-        return Div::tag()
-            ->attributes($this->itemViewAttributes)
-            ->content("\n" . $content)
-            ->encode(false)
-            ->render();
+        return $this->itemViewTag === null
+            ? trim($content)
```

Comment:
```suggestion
            ? $content
```

Reply:
In this case, an extra `\n` is included when the `itemViewTag` is set to `null`. Adding this `trim()` all the tests pass.

Reply:
I think in this case is OK. Extra "\n" appears from item view file.

Reply:
Then I'll remove the `trim()` and change the expected test output